src/main/java/com/puppycrawl/tools/checkstyle/ConfigurationLoader.java

```diff
@@ -270,7 +270,7 @@ public static Configuration loadConfiguration(String config,
     /**
      * Returns the module configurations from a specified input source.
      * Note that if the source does wrap an open byte or character
-     * stream, clients are required to close that stream by themselves
+     * stream, clients are required to close that stream by themselves.
```

Comment:
I disagree with this change if fixing the Check flagged this. I am not saying the change is completely invalid, but it doesn't seem tied to this issue.

https://checkstyle.org/config_javadoc.html#SummaryJavadoc_Description
> Check also violate Javadoc that does not contain first sentence....

The first sentence is "Returns the module configurations from a specified input source." and it ends with a period. The check should not have produced a violation here.

Reply:
ok got it. I have to just look for first sentence in the scope of this issue. 
current code is looking for https://github.com/checkstyle/checkstyle/blob/887aab9402e4e24b2e9c5d7b005e9383c889929e/src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheck.java#L340
period in whole summary so i have also make changes accordingly but i will update this as to see the period only first sentece

Reply:
@rnveach I am converting this pr as a draft right know i am not finding any proper fix for this according to current code 

i also dont understand why this method is name as https://github.com/checkstyle/checkstyle/blob/811983e0f89edf2419b9165ca60bb04a7c3c1ea6/src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheck.java#L718
`getFirstSentence`  it is not giving the first sentence at all it is just returning the string of whole javadoc

Reply:
This method returns first sentence till `periodSuffix`. If you javadoc summary just contains one sentence then it will return the complete javadoc. Try this method with multiple sentences in your javadoc.